OpenEdXMobile/src/main/java/org/edx/mobile/util/JavaUtil.java

```diff
@@ -67,6 +71,31 @@ public static String getDurationString(long duration) {
         return String.format("%02d:%02d:%02d", hours, mins, secs);
     }
 
+    /**
+     * Converts and returns the provided duration into a readable format i.e. hh:mm. Returns null
```

Comment:
please update with.
`Converts and returns the provided duration into a readable format (i.e. 1 hour 55 mins) OR returns null if the duration is zero or negative.`